City-of-Helsinki/helsinki-paatokset
# PP-216: Policymaker page + documents list fixes

Fixes:

- Policymaker document lists are now formed from API-retrieved data. New function PolicymakerService-class.
- Policymaker sidebar is now displayed on subpages, too
- Documents and decisions list blocks have improved frontend performance + simplified structure. Lists now use unified twig templates. 
- Improved PolicymakerService query functions
- Snuck in a sanity check to Helsinki Kanava module
- Did some cleanup on main content styles

To test:

- Run `make drush-cim && make drush-cr`. View policymaker-page (eg. /paattajat/kaupunginvaltuusto) and subpages  (paattajat/*/asiakirjat)
- Verify fixes listed above
